### PR TITLE
Fixes #62

### LINKED ISSUE #62: Linear model example missing final prediction values
linear model using prophet shows the prediction error on the test set but does not shows the final prediction.

### PR DESCRIPTION
**Pull request by jfdelgad**
Fixes #62 .

Changes proposed in this PR:

-  Adds code for final prediction after evaluating the model on the validation set
- 
-

### CODE REVIEW

**Review by trentmc — CHANGES_REQUESTED** [review]
I don't understand the motivation for this.

Before your change: it already trained only on the training data. And it got nmse from the test data. What's the issue with that?

**Comment by trentmc** [comment]
Small comment: on all PRs, make its title also refer to the github issue title. In additional to the github issue number. I usually just copy and paste the github issue title into the PR issue title.

**Comment by jfdelgad** [comment]
> I don't understand the motivation for this.
> 
> Before your change: it already trained only on the training data. And it got nmse from the test data. What's the issue with that?

The prediction made is in on portion of the data. The data available was separated in train and test (which is actually validation as it is used to iterate on variations of the model). But the final prediction should be on future values of dates not present in the data collected. that is what the participants will face when creating a valid submission.

I do not think this is critical but it is more close to reality. Let me know.

**Comment by trentmc** [comment]
Thanks for the clarification.

I get it now. This makes sense. Great job for catching this! Approved.

You'll need to do this in your other readme too, right?

**Review by trentmc — APPROVED** [review]
